mlflow/pytorch/torchserve/deploy/serve.py

```diff
@@ -0,0 +1,388 @@
+        resp = requests.delete(url)
+        if resp.status_code != 200:
+            raise Exception(
+                "Unable to delete deployment for name %s. Server returned status code %s and response: %s"
+                % (name, resp.status_code, resp.content)
+            )
+        return None
+
+    def update_deployment(self, name, model_uri=None, flavor=None, config=None):
+        """
+        Update the deployment with the name given at --name from the specified target
+
+        Using -C or --config additional parameters shall be updated for the corresponding model
+        """
+
+        query_path = ""
+
+        if config is not None:
+            for key in config:
+                query_path += "&" + key + "=" + str(config[key])
+
+            query_path = query_path[1:]
+
+        url = "{}/{}/{}?{}".format(self.management_api, "models", name, query_path)
+        resp = requests.put(url)
+
+        if resp.status_code != 202:
+            raise Exception(
+                "Unable to update deployment with name %s. Server returned status code %s and response: %s"
+                % (name, resp.status_code, resp.content)
+            )
+        return {"flavor": flavor}
+
+    def list_deployments(self):
+        """
+        List the names of all model deployments in the specified target. These names can be used with
+        delete , update and get commands
+        """
+
+        deployment_list = []
+        limit = self.default_limit
+        nextPageToken = 0
+
+        while True:
+            url = "{}/{}".format(self.management_api, "models")
+
+            input_params = {"limit": limit, "next_page_token": nextPageToken}
+
+            resp = requests.get(url, params=input_params)
+            if resp.status_code != 200:
+                raise Exception("Unable to list deployments")
+            temp = json.loads(resp.text)
+            model_count = len(temp["models"])
+            for i in range(model_count):
+                tempDict = {}
+                key = temp["models"][i]["modelName"]
+                tempDict[key] = temp["models"][i]
+                deployment_list.append(tempDict)
+            if "nextPageToken" not in temp:
+                break
+            else:
+                nextPageToken = temp["nextPageToken"]
+        return deployment_list
+
+    def get_deployment(self, name):
+        """
+        Print the detailed description of the deployment with the name given at --name
+        in the specified target
+        """
+
+        url = "{}/{}/{}".format(self.management_api, "models", name)
+        resp = requests.get(url)
+        if resp.status_code != 200:
+            raise Exception(
+                "Unable to get deployments with name %s. Server returned status code %s and response: %s"
+                % (name, resp.status_code, resp.content)
+            )
+        return {"deploy": resp.text}
+
+    def predict(self, deployment_name, df):
```

Comment:
This makes sense at least to start with. In order to support higher-dimensional input, we could (in a follow-up PR) extend this API so that `df` can also be e.g. a dictionary of names to tensors, or a tuple of tensors, etc. It would be good to be use-case driven in our choice there, i.e. do what makes sense to unblock the most common use cases / some of the examples we'd like to add.

Reply:
We will add a separate PR for examples along with ReadMe files.

Reply:
We have added  the support for accommodating tensor as input to the TorchServe deploy plugin. Added 2 examples for mnist - https://github.com/chauhang/mlflow/tree/feature-torchserve-plugin/mlflow/pytorch/torchserve/examples/MNIST . The first one uses cli and takes json as input and the second one uses plugin and takes tensor as input.

Please review and let us know your comments. 

We are working on Bert Sentiment example as well.

Reply:
We have uploaded Bert Sentiment and updated readme files as well.